lesson_10/test_homework10.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from homework_10 import log_event
+
+class MyTestCase(unittest.TestCase):
+
+    name = "Success_name"
+    status = "success"
```

Comment:
Your tests fail in case your status in logs has empty values - it is better to create status as local variable in this case for each test case separately. or each time do this in each test case -> self.status = "desired status". And in assertions use self.status instead of status.

Reply:
no, even if you move variables it does not work